app/models/item.rb

```diff
@@ -0,0 +1,6 @@
+class Item < ApplicationRecord
+
+  validates :title, :description, :price, presence: true
+  validates :title, :description, uniqueness: true
```

Comment:
Looks good! @CjMoore are you thinking we add image validations at some point? When I add categories, should I add a validation for that as well for item?